datafusion/expr/src/udaf.rs

```diff
@@ -574,6 +595,10 @@ pub trait AggregateUDFImpl: Debug + Send + Sync {
     fn is_descending(&self) -> Option<bool> {
         None
     }
+    // Return the value of the current UDF from the statistics
```

Comment:
```suggestion
    /// Return the value of this UDF for the query if it can be determined entirely from
    /// statistics and arguments. 
    ///
    /// For example, if the minimum value of column `x` is known exactly in the statistics, 
    /// then `MIN(x)` can be replaced by that value, significantly improving query performance. 
```